estudios-de-fonetica-experimental.csl

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
```

Comment:
With `default-locale="en-GB"`, this style is currently locked to British English. It looks like this journals publishes in multiple languages (Spanish, Catalan, Galician, French, Italian or English, per http://stel.ub.edu/labfon/sites/default/files/XXVII-14-normas.pdf), so we probably shouldn't lock the style to a particular locale. @wendyelviragarcia, do you agree?

Reply:
I totally agree